hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesMain.java

```diff
@@ -206,6 +205,18 @@ public static void main(final String... args) throws Exception {
         final var recycleBin =
                 RecycleBin.create(metrics, configuration, getStaticThreadManager(), time, fileSystemManager, selfId);
 
+        // Create initial state for the platform
+        final var reservedState = getInitialState(
+                configuration,
+                recycleBin,
+                version,
+                hedera::newMerkleStateRoot,
+                SignedStateFileUtils::readState,
+                Hedera.APP_NAME,
+                Hedera.SWIRLD_NAME,
+                selfId,
+                diskAddressBook);
+
```

Comment:
@thomas-swirlds-labs  Are you planning to move `migrateAndInitialize` from Hedera here in a future PR ?

Reply:
I've moved it here now, thanks @Neeharika-Sompalli !